cmd/status.go

```diff
@@ -0,0 +1,41 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package cmd
+
+import (
+	"os"
+
+	"github.com/dapr/cli/pkg/kubernetes"
+	"github.com/dapr/cli/pkg/print"
+	"github.com/dapr/cli/utils"
+	"github.com/gocarina/gocsv"
+	"github.com/spf13/cobra"
+)
+
+var StatusCmd = &cobra.Command{
+	Use:   "status",
```

Comment:
If it works only in Kubernetes, then I think we can avoid passing "--kubernetes " flag explicitly..

Reply:
/cc @msfussell 

We took a decision a while back to allow support for other platforms in the future, and that's why we have explicit flags for platforms.